From db61989bbb8609ae7a4746303ebf0f4c85d8da95 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 7 Jul 2025 16:07:14 +0200
Subject: [PATCH 2/2] u-boot: Disable EFI capsule and EFI loader

EFI loader only on Turris 1.x, EFI capsule everywhere as it depends on
GnuTLS and as we don't need it, let's disable it instead of adding more
dependencies.
---
 package/boot/uboot-tools/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/package/boot/uboot-tools/Makefile b/package/boot/uboot-tools/Makefile
index 05de9eb7a0..b397aea40c 100644
--- a/package/boot/uboot-tools/Makefile
+++ b/package/boot/uboot-tools/Makefile
@@ -85,6 +85,10 @@ define Build/Configure
 	$(call Build/Compile/Default,tools-only_defconfig)
 	$(SED) 's/CONFIG_TOOLS_LIBCRYPTO=y/# CONFIG_TOOLS_LIBCRYPTO is not set/' $(PKG_BUILD_DIR)/.config
 	$(SED) 's/CONFIG_TOOLS_KWBIMAGE=y/# CONFIG_TOOLS_KWBIMAGE is not set/' $(PKG_BUILD_DIR)/.config
+	$(SED) 's|^CONFIG_TOOLS_MKEFICAPSULE=.|# CONFIG_TOOLS_MKEFICAPSULE is not set|' $(PKG_BUILD_DIR)/.config
+ifeq ($(CONFIG_TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x),y)
+	$(SED) 's|^CONFIG_EFI_LOADER=y|# CONFIG_EFI_LOADER is not set|' $(PKG_BUILD_DIR)/.config
+endif
 endef
 
 MAKE_FLAGS += \
-- 
2.50.0

